lib/quiet_quality/version_control_systems/git.rb

```diff
@@ -41,6 +60,44 @@ def self.default_branch(remote:)
       def comparison_base(sha:, comparison_branch:)
         git.merge_base(comparison_branch, sha).first.sha
       end
+
+      private def changed_lines_for(diff)
```

Comment:
Why the per-method private declarations?

Reply:
I've gotten into the habit of doing it in general at work; given some of the lengthlier classes there it makes it easier to know what's private and what isn't. I suppose it's not necessary here, and adds quite a bit more code... 🔪

Reply:
It's nice to do when you want to co-locate private methods with the places they're used, but I consider that a smell indicating overlarge classes, generally. This one's on the verge of "too large", I'll probably refactor around an operations pattern if we need to add another complex capability in here.